css/cssom-view/scrollIntoView-direction-rtl.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<title>CSSOM View - scrollIntoView considers direction:rtl</title>
+<meta charset="utf-8">
+<link rel="author" title="Cathie Chen" href="mailto:cathiechen@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#dom-element-scrollintoview">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<style>
+.box {
+  float: left;
+  width: 200px;
+  height: 200px;
+}
+#scroller {
+  direction: rtl;
+  overflow-x: scroll;
+  width: 300px;
+  height: 215px;
+}
+#container{
+  width: 600px;
+  height: 200px;
+}
+#target {
+  background-color: #ff0;
+}
+</style>
+<body>
+<div id="scroller">
+  <div id="container">
+    <div class="row">
+      <div class="box"></div>
+      <div class="box" id="target"></div>
+      <div class="box"></div>
+    </div>
+  </div>
+</div>
+
+<script>
+var target = document.getElementById("target");
+var scroller = document.getElementById("scroller");
+var box_width = target.offsetWidth;
+
+// Get the left edge of scrolling area.
+scroller.scrollLeft = -10000;
+var leftEdge = scroller.scrollLeft + box_width;
+// Get the right edge of scrolling area.
+scroller.scrollLeft = 10000;
+var rightEdge = scroller.scrollLeft - box_width;
+
```

Comment:
I think I would prefer explicit values -box_width and -2 box_width + scroller_width so it's clearer that we match what is in the spec.

Since it's a new test, I guess it's ok if chrome does not pass it.

Reply:
Done